pallets/subtensor/src/lib.rs

```diff
@@ -701,6 +701,19 @@ pub mod pallet {
         ValueQuery,
         DefaultStakesPerInterval<T>,
     >;
+
+    #[pallet::storage]
+    /// DMAP ( netuid, hotkey ) --> u64 | Returns the total stake for a hotkey on a specific subnet.
+    pub type TotalSubnetStake<T: Config> = StorageDoubleMap<
```

Comment:
I can see you insert the data to the new storage when epoch update. But the storage is never used.